src/bindings/io/test/test_utils.c

```diff
@@ -0,0 +1,107 @@
+/**
+ * @file
+ *
+ * @brief Tests for haskelltemplate plugin
```

Comment:
?

Reply:
Ops...must have accidentially copied that along with the license header.